src/Engine/Util/GPUProfiling.cpp

```diff
@@ -0,0 +1,42 @@
+#include "GPUProfiling.hpp"
+
+#include <assert.h>
+#include "Engine/Manager/Managers.hpp"
+#include <Utility/Log.hpp>
+
+GPUProfiling::GPUProfiling(const std::string& name, Video::Query::Type type) : active(false) {
+    // Check if profiling.
+    if (Managers().profilingManager->Active()) {
+        active = true;
+
+        // Get type.
+        switch (type) {
+        case Video::Query::TIME_ELAPSED:
```

Comment:
Perhaps indent case labels. Not sure what style guides say about this.

Reply:
Fixed